_posts/2020-07-3-status-update.md

```diff
@@ -0,0 +1,78 @@
+layout: post
+title: "3th July 2020: IAC Team Status Update"
+author: daianamezdrea
+categories:
+  - team
+  - status
+tags:
+  - releases
+  - community_management
+  - java_ks
+  - tagmail
+  - ibm_installation_manager
+  - litmus
+  - shoutouts
+---
+
+The IAC team has been working on a lot of things over the past week.
+We want to share with our community some of the most interesting things.
+Here’s a brief overview of what we’ve been up to:
+
+## Module Releases from IAC Team
+
+- [puppetlabs/java_ks](https://github.com/puppetlabs/puppetlabs-java_ks)(3.2.0)
+- [puppetlabs/tagmail](https://github.com/puppetlabs/puppetlabs-tagmail)(3.3.0)
+- [puppetlabs/ibm_installation_manager](https://github.com/puppetlabs/puppetlabs-ibm_installation_manager)(2.2.0)
+
+## puppet_litmus 0.18.4 was released
+
+We have released v0.18.4 of the [puppet_litmus gem][https://github.com/puppetlabs/puppet_litmus].
+There were quite a number of fixes but a few highlights worth mentioning:
+
+[DavidS](https://github.com/DavidS) - Protect version reporting from undefined-ness: [312](https://github.com/puppetlabs/puppet_litmus/pull/312) 
+
+[lswith](https://github.com/lswith) - Ignore stderr of serverspec commands by setting request pty to false [309](https://github.com/puppetlabs/puppet_litmus/pull/309)
+
+## Community Shout outs for IAC
+
+We think that is no doubt about our community contributors, they are always ready to help and they have excellent ideas. Here are some highlights of this week:
+
+Thanks to [Guillaume Leroy](https://github.com/leroyguillaume) for the bug [fix](https://github.com/puppetlabs/puppetlabs-tomcat/pull/400) on tomcat module.
+
+Thanks to [Marc Schöchlin](https://github.com/scoopex) for the kubernetes PRs which bring a lot of new changes.
+
+## Customer tickets closed this week
+
+The team have closed out the following customer tickets: MySQL issue (MODULES-10597) and a vcsrepo issue (MODULES-10659).
```

Comment:
```suggestion
The team have closed out the following customer tickets: MySQL issue ([MODULES-10597](https://tickets.puppetlabs.com/browse/MODULES-10597)) and a vcsrepo issue ([MODULES-10659](https://tickets.puppetlabs.com/browse/MODULES-10659)).
```

Reply:
My bad here, I forgot to add the links ... Daiana--